theano/gpuarray/fp16_help.py

```diff
@@ -2,20 +2,40 @@
 
 
 def work_dtype(dtype):
+    """
+    Return the data type for working memory.
+
+    """
     if dtype == 'float16':
         return 'float32'
     else:
         return dtype
 
 
 def load_w(dtype):
+    """
+    Return the function name to load data.
+
+    This should be used like this::
+
+        code = '%(load_f)s(ival)' % (load_w(input_type),)
```

Comment:
this is not valid python code. I think you mean:

    code = '%s(ival)' % (load_w(input_type),)